dot/network/utils.go

```diff
@@ -186,17 +183,12 @@ func readLEB128ToUint64(r io.Reader, buf []byte) (uint64, int, error) {
 }
 
 // readStream reads from the stream into the given buffer, returning the number of bytes read
-func readStream(stream libp2pnetwork.Stream, bufPointer *[]byte, maxSize uint64) (int, error) {
+func readStream(stream libp2pnetwork.Stream, bufPointer *[]byte, maxSize uint64) (tot int, err error) {
```

Comment:
opinion/preference: better to not use named returns to avoid returning unexpectedly by mistake

Reply:
That makes sense, these changes were from @EclesioMeloJunior (I'm not sure why they're showing as changes on this PR, I think I may have done something wrong when rebasing this branch onto feat/parachain.)